client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -172,12 +172,6 @@ export class CommentNavigation extends Component {
 					) }
 				</NavTabs>
 
-				<CommentNavigationTab className="comment-navigation__actions comment-navigation__open-bulk">
```

Comment:
True, but a feature flag will come in handy if we want to develop the bulk actions in smaller steps while the m1 is live.